transports/tls/src/certificate.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 Parity Technologies (UK) Ltd.
+// Copyright 2022 Protocol Labs.
```

Comment:
Just removing the copyright doesn't look right to me. I'm not a lawyer, but the original copyright should stay?

Reply:
That was an oversight, thanks for pointing that out!